errors.md

```diff
@@ -46,7 +46,6 @@ The following errors occur when something goes wrong while converting the SQL qu
 |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [Error 3002](#error-3002) | A SELECT list with multiple values cannot contain a non-namespaced `*` (i.e., `SELECT a, *, b FROM myTable` is not supported). A non-namespaced `*` must be used by itself.                                  |
 | [Error 3004](#error-3004) | The array data source contains an identifier. Array data sources must be constant.                                                                                                                           |
-| [Error 3005](#error-3005) | SELECT DISTINCT is not allowed.                                                                                                                                                                              |
```

Comment:
Good catch updating this! It reminded me to do the same on #21. There is also a longer description of the error down on line 222 we should remove.